OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -183,7 +183,7 @@ protected virtual AttackStatus TickAttack(Actor self, AttackFrontal attack)
 			maxRange = armaments.Min(a => a.MaxRange());
 
 			var pos = self.CenterPosition;
-			var mobile = move as Mobile;
+			var mobile = (Mobile)move;
```

Comment:
I'm pretty sure this will crash if `move` is not an instance of `Mobile`. Are we sure it is always `Mobile` here and cannot be `Aircraft` instead for example?

Reply:
Possibly, I also ran into a crash with one of the Locomotor changes of that commit, so it's probably better to just drop it.